summary/Readme.rst

```diff
@@ -1,3 +1,10 @@
+Requirements
+------------
+
+This plugin requires beautifulsoup4 to ensure correctness of the extracted markup::
+
+    pip install pillow beautifulsoup4
```

Comment:
I do not think `pillow` is needed.

Reply:
It totally isn't, copy/paste failure. I'll fix that when I get a moment later today